### PR TITLE
enable checking message dependencies

### PR DESCRIPTION
**Pull request by dirk-thomas**
Connects to ros2/rosidl#12

@esteve @tfoote @wjwwood Please review.

### CODE REVIEW

File: rosidl_typesupport_introspection_cpp/package.xml [hunk 1]
```diff
@@ -14,6 +14,8 @@
   <buildtool_export_depend>ament_cmake</buildtool_export_depend>
   <buildtool_export_depend>rosidl_cmake</buildtool_export_depend>
 
+  <exec_depend>rosidl_parser</exec_depend>
```
Comment (tfoote) [inline rosidl_typesupport_introspection_cpp/package.xml:18]: need rosidl_cmake? It's added as an import in https://github.com/ros2/rosidl_dds/pull/19/files#diff-118a9468a832dd2591da76a4bd5a2d54R19

Comment (dirk-thomas) [inline rosidl_typesupport_introspection_cpp/package.xml:18]: Indeed, good catch!

**Comment by tfoote** [comment]
+1

**Comment by wjwwood** [comment]
+1

**Pull request merged**